packages/react-card/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fluentui/react-card",
   "version": "9.0.0-alpha.0",
-  "private": true,
+  "private": false,
```

Comment:
are we sure that we want to release immediately ?  are we in a rush to deliver this to partners ?

Reply:
as you said it's not targeted for the beta release

Reply:
Partner's needs are basically ASAP, hence the need for this first phase iteration.

Reply:
Another note: if partners consume this as a separate package along with react-components there would be compatibility issues since the removal of `mergeProps` and other breaking changes to slots. It would be hard for them to figure out why things are breaking because we use prerelease tags that don't communicate breaking changes